client/src/components/home/home.styles.scss

```diff
@@ -1,63 +1,135 @@
 .home-container{
-    padding-top:10%;
-}
-.button-container {
-    display:flex;
-    justify-content: center;
-    align-items: center;
-    flex-direction: column;
+    padding-top: 10%;
+    .main-container{
+        display:flex;
+        justify-content: center;
+        flex-direction: column;
+        padding: 3% 2% 3%;
+        margin-top: 1%;
+        margin-bottom: 0.5%;
+        border-bottom: 1px solid rgb(232, 232, 232);
+        border-top: 1px solid rgb(232, 232, 232);
+        border-left: 0.2px solid rgb(232, 232, 232);
+        border-right: 0.2px solid rgb(232, 232, 232);
+        background: rgb(255,255,255);
+        border-bottom-left-radius: 20px;
```

Comment:
Can't we replace those 4 lines with same values with just `border-raduis: 20px`?